test/sqllogictest/cockroach/case_sensitive_names.slt

```diff
@@ -234,7 +234,7 @@ query TT
 SHOW CREATE VIEW "YV"
 ----
 materialize.public.YV
-CREATE VIEW "materialize"."public"."YV" AS SELECT "x", "Y" FROM "materialize"."public"."foo"
+CREATE VIEW "materialize"."public"."YV" AS SELECT "x", "Y" FROM [u5 AS materialize.public.foo]
```

Comment:
Does this mean that ```SHOW CREATE VIEW``` is no longer be re-executable outside of the current database instance? I mean if I am migrating my views to another instance, ```u5``` will be a totally different object on that instance or may not exist at all.

Reply:
Ah yeah, it probably does. I believe we should fix this by teaching SHOW CREATE VIEW to parse this SQL that you see here and then reserialize the string using logic that emits the name directly rather than as an ID.

Reply:
Hm, that's a problem though, right? Because the goal all along was to avoid doing that? We could actually implement the logic to do this, I guess, but I think it's pretty subtle?